ietf-optical-impairment-topology.tree

```diff
@@ -49,39 +56,110 @@ module: ietf-optical-impairment-topology
                 +--:(concentratedloss)
                    +--ro concentratedloss
                       +--ro loss?   decimal64
-  augment /nw:networks/nw:network/nw:node/tet:te/tet:tunnel-termination-point:
+  augment /nw:networks/nw:network/nw:node/tet:te
+            /tet:tunnel-termination-point:
     +--ro OTSiG-element* [OTSiG-identifier]
     |  +--ro OTSiG-identifier    int16
     |  +--ro OTSiG-container
     |     +--ro OTSi* [OTSi-carrier-id]
     |        +--ro OTSi-carrier-id           int16
+    |        +--ro transponder-ref?          leafref
+    |        +--ro transceiver-ref?          leafref
+    |        +--ro configured-mode?          leafref
     |        +--ro OTSi-carrier-frequency?   decimal64
     |        +--ro OTSi-signal-width?        decimal64
```

Comment:
Do we need this attribute?

Reply:
These two attributes represents nominal frequency and width of the signal .

Reply:
Take the attribute but move it in the common grouping